tests/data/3.extracted_features/gait/gait_by_minute_meta.json

```diff
@@ -0,0 +1,44 @@
+{
+    "study_id": "PPP",
+    "device_id": "Verily Study Watch",
+    "subject_id": "X",
+    "ppp_source_protobuf": "WatchData.IMU.Week104.raw",
+    "metadata_version": "0.1",
+    "start_iso8601": "2021-06-27T16:52:20Z",
+    "end_iso8601": "2021-06-27T17:04:30Z",
+    "rows": 13,
+    "endianness": "little",
+    "data_type": "float",
+    "bits": 64,
```

Comment:
Why is this not the same as the data in the method? 
` metadata_time.bits = 32`